Backend/Backend/Infrastructure/FileStorageHelper.cs

```diff
@@ -60,6 +62,26 @@ public enum TranscriptStatus
     };
 
     #region User Profile
+    
+    public static bool ValidateAvatar(IFormFile? file, out object? errorObject) {
+        errorObject = null;
+        if (file is not null) {
+            string extension = Path.GetExtension(file.FileName);
+            bool ok = false;
```

Comment:
c# has a .Contains() Method, you dont need to do this foreach loop

Reply:
something like this 
            string test = ".gif";
            bool ok = AVATAR_EXTENSIONS.Contains(test);